conf/application.conf

```diff
@@ -58,27 +47,147 @@ contact-frontend {
 
 devHubBaseUrl = "http://localhost:9680"
 
-microservice {
-  services {
-    auth {
-      host = localhost
-      port = 8500
+wiremock-port = 11111
+wiremock-port = ${?WIREMOCK_PORT}
+
+
+Stub {
+  auditing {
+    enabled=false
+  }
+
+  microservice {
+    services {
+      auth {
+        host = localhost
+        port = 8500
```

Comment:
The Auth service has not been implemented in the API Service Stub so we are using the standard 'sm --start AUTH' which will serve on port 8500.

We could look at extending the API Service Stub to implement the necessary entry points as a separate JIRA.